assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/requirements.txt

```diff
@@ -5,7 +5,7 @@ azureml-acft-contrib-hf-nlp=={{latest-pypi-version}}
 azureml-acft-multimodal-components=={{latest-pypi-version}}
 # needed by multimodal
 mltable=={{latest-pypi-version}}
-datasets==2.12.0
+datasets==2.14.7
```

Comment:
If we have it in DockerFile, we can remove it from requirements.txt for now. 
Introduce it later once the dependencies are resolved.